apis/sgv2-quarkus-common/src/main/resources/application.yaml

```diff
@@ -1,7 +1,148 @@
+---
+# please keep properties in the alphabetical order
```

Comment:
Maybe add a note that these are the defaults for all APIs, and changes here will affect all APIs (except if overridden by API's application.yaml)